android/autodispose-androidx-lifecycle-test/gradle.properties

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-POM_NAME=AutoDispose (Android Architecture Components Extensions)
-POM_ARTIFACT_ID=autodispose-android-archcomponents
+POM_NAME=AutoDispose (androidx-lifecycle test extensions)
```

Comment:
My capitalizing OCD has trouble with `test extensions`

Reply:
Sorry, went for consistency here

Reply:
"Androidx-Lifecycle" would be more wrong

Reply:
Yeah i know. Not suggesting we change, but just had to point out :D